src/xrGame/player_hud.cpp

```diff
@@ -664,6 +664,7 @@ void player_hud::update_inertion(Fmatrix& trans)
 
         // clamp by PI_DIV_2
         Fvector last;
+        last.set(0,0,0);
```

Comment:
It's better to delete this and use another solution
```suggestion
```
Initialization happens one line below:
```cpp
last.normalize_safe(st_last_dir);
```
But GCC removes this line because `st_last_dir` is static:
```cpp
static Fvector st_last_dir = {0, 0, 0};
```

Probably we can make it volatile and GCC won't remove it:
```cpp
volatile Fvector st_last_dir = {0, 0, 0};
```
`st_last_dir` is used three times in the function.

What do you think guys?